blog-posts/sls-security-headers/sls-security-headers.md

```diff
@@ -1,10 +1,96 @@
 ---
 published: false
-title: 'Why you should not use S3 to publish your static website'
+title: 'Leverage AWS CloudFront managed response headers policies'
 cover_image: 'https://dev-to-uploads.s3.amazonaws.com/uploads/articles/xgzowe15czab4awhanxd.png'
 description: ''
 tags: cloudfront, security, AWS, audit
 canonical_url:
 ---
 
-#
+– Did you just say security headers?
+
+– Yes.
+
+– Are you talking about CORS headers?
+
+– No.
+
+# HTTP security headers: a very simple addition to your website to efficiently prevent attacks
+
+I have learnt about HTTP security headers very late in my full stack developer carrier. In the last 5 years, have worked in a dozen teams to develop and run web applications.
+
+This is only very recently, when I joined [Theodo](https://www.theodo.fr/), that I was explained what HTTP security headers actually are. 😮
+
+HTTP security headers are seemingly unknown to many full stack developers.
+
+HTTP security headers are a mechanism to **protect your application from common attacks**. HTTP security headers are, as their name suggests, a set of HTTP headers with security purposes.
+
+The table below shows a subset of HTTP security headers. The value is strongly recommended. The description mentions the common attacks that the header protects against.
```

Comment:
Je trouve pas ce morceau de phrase super clair: 'The value is strongly recommended.'.

Je renommerai bien la colone dans ton tableau 'Recommended Value' plutot, et mettrai une phrase dans le genre de 'The description mentions the common attacks that the header protects against. The recommended value column contains optimal header values to protect you against these threats.'